apps/dashboard/src/lib/components/Course/components/Lesson/Exercise/ViewMode.svelte

```diff
@@ -391,19 +291,25 @@
     </div>
   </RoleBasedSecurity>
 {:else if $questionnaireMetaData.isFinished}
-  {#if isLoading}
-    <GradingLoader />
-  {:else if !isLoading && !isLoadingAutoSavedData}
+  {#if !isLoadingAutoSavedData}
     <div class="flex items-center justify-between">
       <div class="flex flex-col lg:flex-row items-start lg:items-center lg:space-x-4 w-full">
         <h2 class="text-xl font-normal">{$questionnaire.title}</h2>
+
         {#if STATUS.GRADED === $questionnaireMetaData.status}
           <span
             class="status-text bg-green-700 text-white py-1 px-2 text-center"
-            title="Status: Pending Review"
+            title="Status: Review completed"
           >
             Graded
           </span>
+        {:else if $course.type === COURSE_TYPE.SELF_PACED}
+          <span
+            class="status-text bg-green-700 text-white py-1 px-2 text-center"
+            title="Status: Submitted"
+          >
+            Submitted
+          </span>
```

Comment:
I noticed in this file we don't support translation for some text.

please can you go over it and make sure we have translation in this file